Classes/Base/Config.php

```diff
@@ -52,10 +52,12 @@ public static function get( array $paths, array $local = array() )
 			self::$config = $conf;
 		}
 
-		if( isset( $local['typo3']['tsconfig'] ) )
+		// Check for array
+		if( isset( $local['typo3']['tsconfig'] ) && is_array( $local['typo3']['tsconfig'] ) )
 		{
 			$tsconfig = \Aimeos\Aimeos\Base::parseTS( $local['typo3']['tsconfig'] );
-			$local = \TYPO3\CMS\Extbase\Utility\ArrayUtility::arrayMergeRecursiveOverrule( $local, $tsconfig );
+			// merge tsconfig to local, ignore empty values
+			$local = \TYPO3\CMS\Extbase\Utility\ArrayUtility::arrayMergeRecursiveOverrule( $local, $tsconfig, false, false );
```

Comment:
To not override the loaded settings by empty ones of tsconfig the last parameter has to be false